ext/pdo_pgsql/tests/bug70861.phpt

```diff
@@ -41,6 +41,8 @@ $stmt->execute();
 
 fclose($f);
 
+$db->exec('DROP TABLE test_blob_crash');
```

Comment:
So for those sorts of clean-ups we use the ``--CLEAN--`` section, as it is run after every test regardless of success or failure or if the test crashed.

Also realising that those tests currently cannot run in parallel as they use the same DB table _sigh_ but that's another issue.